src/components/Button/Button.tsx

```diff
@@ -0,0 +1,73 @@
+/** @jsxImportSource @emotion/react */
+import React from 'react';
+import styled from '@emotion/styled';
+
+// ButtonProps 인터페이스 정의
+export interface ButtonProps {
+  theme: 'primary' | 'secondary'; // 버튼의 테마 (primary 또는 secondary)
+  size: 'small' | 'medium' | 'large' | 'responsive'; // 버튼의 크기 (small, medium, large 또는 responsive)
+  children: React.ReactNode; // 버튼 내부의 내용
+  onClick: () => void; // 버튼 클릭 시 호출될 함수
+}
+
+// styled.button을 사용하여 버튼 스타일 정의
+const Button = styled.button<ButtonProps>`
+  border: none;
+  cursor: pointer;
+  font-size: ${(props) => {
+    switch (props.size) {
+      case 'small':
+        return '0.75rem'; // 작은 버튼의 글꼴 크기
+      case 'large':
+        return '1.25rem'; // 큰 버튼의 글꼴 크기
+      case 'responsive':
+        return '1rem'; // 반응형 버튼의 글꼴 크기
+      default:
+        return '1rem'; // 기본 글꼴 크기
```

Comment:
소소하게 아래처럼도 작성가능하니 참고하시면 좋을거 같아요

```suggestion
      case 'responsive':
      default:
        return '1rem'; // 기본 글꼴 크기
```